hikari/impl/rest.py

```diff
@@ -1204,26 +1204,32 @@ async def fetch_message(
         assert isinstance(response, dict)
         return self._entity_factory.deserialize_message(response)
 
-    async def _create_message(
+    def _build_message_payload(  # noqa: C901
         self,
-        route: routes.CompiledRoute,
-        body: data_binding.JSONObjectBuilder,
-        query: typing.Optional[data_binding.StringMapBuilder] = None,
         *,
```

Comment:
Just for the sake of it
```suggestion
        /,
        *,
```